modin/data_management/functions/groupby_function.py

```diff
@@ -148,10 +145,13 @@ def caller(
         numeric_only=True,
         **kwargs,
     ):
-        if not (isinstance(by, (type(query_compiler)) or hashable(by))) or isinstance(
+        groupby_args = groupby_args.copy()
+        if not (isinstance(by, type(query_compiler)) or hashable(by)) or isinstance(
             by, pandas.Grouper
         ):
             by = try_cast_to_pandas(by, squeeze=True)
+            if by is not None:
+                by = wrap_into_list(by, groupby_args.pop("level", []))
```

Comment:
the same as [here](https://github.com/modin-project/modin/pull/2790/files#r584803662)